cypress/support/step_definitions/index.ts

```diff
@@ -58,19 +67,45 @@ When('I wait for {int} milliseconds/ms', function (ms: number) {
   cy.wait(ms)
 })
 
-When(/^I "(.*)"(.*)? on the "(.*)" element$/, (event: string, object: string | number | undefined, selector: string) => {
+When(/^I "(.*)" the "(.*)" element(?: and select "(.*)".*)?$/, (event: string, selector: string, value?: string) => {
   switch (event) {
-    case 'select':
-      if (typeof object === 'undefined') {
-        throw new Error()
+    case 'check':
```

Comment:
Did we say we were gonna remove the "check" and just use "click"? Seeing as you "click" on the checkbox or in a lot of cases "click" on the label? Not sure, not super important this, just asking in case we did say that.

Reply:
I don’t care much for the difference so I’ve removed it, making `event` no longer variable so I removed that, too.